src/test/hammerdb/run_hammerdb.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# fail if trying to reference a variable that is not set.
+set -u
+# exit immediately if a command fails
+set -e
+# echo commands
+set -x
+
+rg=$1
+
+export RESOURCE_GROUP_NAME="${rg}"
+export AZURE_REGION=westus2
```

Comment:
Minor: why westus2? I thought eastus2 has more capacity?

Reply:
because westus2 seems to be fastest :D I think as we are comparing two branches in the same cluster (master vs that branch) we can just remove this as well to decide the region randomly. But as we cannot create more than 2 resource groups at a time with these jobs ( even 2 at the same time will be very rare) capacity won't be that problem maybe?